packages/dds/tree/api-extractor/api-extractor-lint-legacy-beta.esm.json

```diff
@@ -0,0 +1,5 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "<projectFolder>/../../../common/build/build-common/api-extractor-lint.entrypoint.json",
+	"mainEntryPointFilePath": "<projectFolder>/dist/legacy.d.ts"
```

Comment:
These files should get reverted but note that this is the wrong path for `esm` - this would need to be `lib` and not `dist` to be correct. I doubt this is enough to explain the fluid-build errors you were seeing, but we should remove these either way.

Reply:
There shouldn't be any differences between this package and the others. This package already has `api-extractor-lint.legacy.esm.json` and `api-extractor-lint.legacy.cjs.json`, and there are corresponding lint scripts.

Reply:
Still building seems like I deleted them based on the last comment
```
deleted:    packages/dds/tree/api-extractor/api-extractor-lint-legacy-beta.cjs.json
deleted:    packages/dds/tree/api-extractor/api-extractor-lint-legacy-beta.esm.json
```